quipucords/api/inspectresult/model.py

```diff
@@ -13,20 +13,17 @@
 from constants import DataSources
 
 RAW_QUERY_POSTGRES = """
-with raw_agg as (
-    select
-        ir.id,
+with raw_agg(ir_id, inspect_group_id, raw_facts) as (
+	select
+		ir.id,
```

Comment:
I know this is a raw string being passed to SQL that doesn't follow Python coding conventions, but may we choose spaces instead of tab characters for these for sake of consistency? (I ran into this exact same line earlier today and was meaning to ask about this anyway!)

```suggestion
    select
        ir.id,
```

Reply:
wait, I'm using tab here? lol
fixing. thanks for spotting it

Reply:
Also the lines below that use raw tab characters. Using spaces makes it easier to copy-paste into a DB shell when manually debugging code too since tabs can trigger other behaviors like autocomplete.